src/components/Review/styles.ts

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components'
+
+export const Review = styled.li`
+  list-style: none;
+  background-color: #E9F4FE;
+  max-width: 400px;
+  padding: 20px;
+  border-radius: 4px;
+  transition: transform .7s;
+
+  &:hover{
+    transform: scale(1.2);
+    z-index: 1;
+    box-shadow: 2px 2px 2px 2px rgba(0, 0, 0, 0.2);
+  }
+  `
+
+export const TextReview = styled.article`
```

Comment:
Gostei do uso `article` , mas eu talvez faria umas mudanças na sua estrutura. Pois você deixou só o texto dentro do article, sendo que o avatar/nome também deveria estar juntos para compor 1 artigo em si.

Mesmo asim, nesse caso, eu também usaria blockquote, exatamente pois é o que semânticamente esse elemento é.

https://css-tricks.com/quoting-in-html-quotations-citations-and-blockquotes/